### PR TITLE
fix(listview): hide actions column if no actions are allowed

### PR DESCRIPTION
**Pull request by nytai**
### SUMMARY

- see title
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

before:
Screen Shot 2020-10-19 at 8 45 33 PM
after:
Screen Shot 2020-10-19 at 8 44 40 PM

### TEST PLAN

- create a user with no edit, delete, and export permissions for dashboards
- sign in as user
- navigate to dashboards list 
- 👀 no actions column

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/dataViewCommon/TableCollection.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ interface TableCollectionProps {
 }
 
 export const Table = styled.table`
+  background-color: ${({ theme }) => theme.colors.grayscale.light5};
```
Comment (nytai) [inline superset-frontend/src/components/dataViewCommon/TableCollection.tsx:37]: looks like this was removed with https://github.com/apache/incubator-superset/commit/820fa473a7fc7571036b86a99ad66396d45e4c2f#diff-e3cf6caa34dff868437ff828d41d0495d30a9701c28d3864c9102a47bf842d22L37

Comment (nytai) [reply superset-frontend/src/components/dataViewCommon/TableCollection.tsx:37]: @kgabryje is this going to affect your use case?

Comment (kgabryje) [reply superset-frontend/src/components/dataViewCommon/TableCollection.tsx:37]: It changes the colour of Query Search table - originally it was transparent, not white. Maybe you could override the background colour just for Query Search?

Comment (nytai) [reply superset-frontend/src/components/dataViewCommon/TableCollection.tsx:37]: sounds good. I think we will soon be replacing that component with a listview component

**Review by riahk — APPROVED** [review]

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**